packages/react-query-typechain/.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Package to npmjs
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      # Add environment variables here
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      - run: npm run build
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist/
+      - name: Publish to NPM
+        run: npm publish --access public
```

Comment:
This action will need to be updated to work in the monorepo, tacklling publishing in a future PR.